src/DependencyInjection/Configuration.php

```diff
@@ -172,7 +172,7 @@ private function addMailAggregatorSection(ArrayNodeDefinition $rootNode): void
                     ->children()
                         ->arrayNode('routing_keys')
                             ->info('Defines the rules that let the MailConsumer listen to routes.')
-                            ->defaultValue(['em_mails_*'])
+                            ->defaultValue(['em_mails.*.*'])
```

Comment:
You can update the `scalarPrototype()->info()` message : 
```
->info('Rule to define routes to listen to. i.e "em_mails.*.*" ...)
```

Reply:
fixed, thanks 👍